src/nvim/ex_cmds2.c

```diff
@@ -392,12 +392,12 @@ int dbg_check_skipped(exarg_T *eap)
  * This is a grow-array of structs.
  */
 struct debuggy {
-  int dbg_nr;                   /* breakpoint number */
```

Comment:
Yep, I thought changing just one struct's comment style makes things inconsistent, and changed all structs and enums in one go. Figured it's a relatively small additional noise but is okay because it keeps things consistent.